simpleviewinc/tap-events-force
# feat(booking-button): Update to match mock-ups

**Ticket**: [ZEN-628](https://jira.simpleviewtools.com/browse/ZEN-628)

## Context
* Unfortunately the width of the buttons on the mock-ups page are not consistent
  * Based on which button you click on, the width and text font size might change
  * So I updated it to work with what I think is best base on the content to be displayed
    * [Looks like this](https://i.imgur.com/xCldPf0.png)
  * There are some edge cases when the button text splits one of the words
    * This is only shown in one place on the mock-up
    * In another location, the text does not split
    * [See here](https://i.imgur.com/fD5vUrn.png)
    * So I set styles based on what worked best across the board


## Updates
* `src/components/button/bookingButton.js`
  * Cleaned up styles
* `src/components/grid/gridRowContent.js`
  * Updated to use `reStyle` where possiable
* `src/theme/components/button/evfButton.js`
  * Added some minor style updates based on changes
* `src/theme/components/grid/gridItem.js` && `src/theme/components/grid/gridRowContent.js` && `src/theme/components/grid/index.js`
  * Removed unused styles

## Tests
**Setup**
* Run the image =>  `keg evf pack run evf:zen-628-mobile-select-btn`
* Navigate it in your browser => http://evf-zen-628-mobile-select-btn.local.keghub.io/

**Test 1**
* Ensure the `Booking Button` look correct following the mockup
* Ensure `On Waiting List` text only breaks once according to the mockups
  * [Mock-ups](https://i.imgur.com/GjeTSw0.png)
  * [Looks like this](https://i.imgur.com/YHOqYhP.png)
